packages/schematics/angular/migrations/update-14/angular-packages-version-prefix.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Rule } from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import { NodeDependencyType } from '../../utility/dependencies';
+import { JSONFile } from '../../utility/json-file';
+
+const PACKAGES_REGEXP = /^@(?:angular|nguniversal|schematics|angular-devkit)\/|^ng-packagr$/;
+
+export default function (): Rule {
```

Comment:
A doc comment for this rule, maybe with a short example, would be really useful to communicate exactly what the purpose of the migration this.